join_github_app/main/routes/auth.py

```diff
@@ -64,11 +64,11 @@ def callback():
         return redirect("/auth/server-error")
 
     org_selection = session.get("org_selection", [])
-    original_email = session.get("email")
+    # user_input_email = session.get("user_input_email")
+    auth0_email = session["user"].get("userinfo", {}).get("email")
```

Comment:
Might be worth not using the `.get()` here 🤔 The user session is populated above, so ideally has this information ✅ 

If it doesn't, we'd prefer it to throw the exception - whereas at the moment the logic will provide a default `None` value - that will get passed to the `send_github_invitations` function (where it may then complain 🚒 )

But better to fail fast! 🚤 🔥 and have it throw the exception on this line, then it's a bit more clear where the problem is actually occurring 😁